src/redisai.c

```diff
@@ -568,7 +567,8 @@ int RedisAI_ModelRun_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
     if (RedisModule_IsKeysPositionRequest(ctx)) {
         return RedisAI_ModelRun_IsKeysPositionRequest_ReportKeys(ctx, argv, argc);
     }
-    return ProcessRunCommand(ctx, argv, argc, CMD_MODELRUN, REDISAI_DAG_WRITE_MODE);
+    bool ro_dag = false;
+    return RedisAI_ExecuteCommand(ctx, argv, argc, CMD_MODELRUN, ro_dag);
```

Comment:
you don't have to declare the variable - you can just send it in the function
leave or modify at your discretion.